locales/en.yaml

```diff
@@ -466,4 +470,6 @@ make_private: Make private
 bot_kicked: Bot was kicked from chat, must be member and admin
 chat_not_found: Chat not found, are you sure you added the bot to the group/channel?
 not_member: You are not member of that chat
-upgraded_to_supergroup: The chat was upgraded to 'supergroup' and the Id has changed, check the chat Id again
\ No newline at end of file
+upgraded_to_supergroup: The chat was upgraded to 'supergroup' and the Id has changed, check the chat Id again
```

Comment:
When we add new locales, we need to do it in all languages because if a user go to the message that uses this new key in a language where there is no value the bot will fail

Reply:
no need to translate it, you can write it in english